docs/website/versioned_docs/version-3.0.0/command-reference/odo-create-namespace.md

```diff
@@ -0,0 +1,32 @@
+---
+title: odo create namespace
+sidebar_position: 3
+---
+
+`odo create namespace` lets you create a namespace/project on your cluster.
+
+To create a namespace you can run `odo create namespace <name>`:
+```shell
+odo create namespace mynamespace
+```
+
+Example - 
+```shell
+odo create namespace mynamespace
+ ✓  Namespace "mynamespace" is ready for use
+ ✓  New namespace created and now using namespace: mynamespace
+```
+
+Optionally, you can also use `project` as an alias to `namespace`.
```

Comment:
Please explain that using Project or Namespace won't change the resource created in the cluster: only the existence of the Project resource type in the cluster will.